src/main/java/org/sagebionetworks/bridge/udd/dynamodb/DynamoHelper.java

```diff
@@ -86,12 +95,33 @@ public StudyInfo getStudy(String studyId) {
     }
 
     /**
+     * Gets the set of survey table IDs for a given study.
+     *
+     * @param studyId
+     *         ID of study to get survey tables
+     * @return set of survey table IDs, may be empty, but will never be null
+     */
+    public Set<String> getSynapseSurveyTablesForStudy(String studyId) {
```

Comment:
StudyIdentifier? Depends on where you call it from, but if you have it, I would pass it along for type safety.